### PR TITLE
Make curl fail helpfully and consistently

### PR DESCRIPTION
**Pull request by andytson-inviqa**
Using full option names to make it clear what they do, and use --location except for on unsupported request methods

### CODE REVIEW

File: src/_base/harness/scripts/mutagen.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ install_mutagen()
         return 1
     fi
 
-    run curl -L -q -sS -f "$download_url" -o .my127ws/utilities/mutagen/mutagen.tar.gz
+    run curl --fail --silent --show-error --location --disable --output .my127ws/utilities/mutagen/mutagen.tar.gz "$download_url"
```
Comment (andytson-inviqa) [inline src/_base/harness/scripts/mutagen.sh]: I'm not sure -q was intentionally used to mean --disable (disables use of .curlrc), or mistaking for quiet?

Comment (hgajjar) [reply src/_base/harness/scripts/mutagen.sh]: Mistaken for quiet 🤦 
I like the full flag naming instead of abbreviations 👍

**Review by hgajjar — DISMISSED** [review]
Looks good to me!

File: src/_base/harness/scripts/mutagen.sh (hunk 1 above)
Comment (andytson-inviqa) [reply src/_base/harness/scripts/mutagen.sh]: yes, it's generally recommended to use long options for scripting, and only use short options for personal usage

**Review by kierenevans — APPROVED** [review]

**Pull request merged**